spec/unit/trello_wrapper_spec.rb

```diff
@@ -47,6 +47,36 @@
     end
   end
 
+  describe '#backup' do
+    it 'raises an error for empty board id' do
+      expect { subject.backup("") }.to raise_error(TrolloloError)
+    end
+
+    it 'raises an error for nil board id' do
+      expect { subject.backup(nil) }.to raise_error(TrolloloError)
+    end
```

Comment:
Differently from `backup`, `retrieve_board_data` actually does something with the value returned (i.e. convert to a hash), that's why I have included that case. But maybe the correct course of action instead would be to assert that an error is raised because the returned value could not be translated to a hash.

As you said, the granularity here may be too small, indeed.